src/particles.cpp

```diff
@@ -589,7 +670,10 @@ void ParticleManager::addNodeParticle(IGameDef* gamedef, scene::ISceneManager* s
 		false,
 		texture,
 		texpos,
-		texsize);
+		texsize,
+		8464,
```

Comment:
Oh looks like the value comes from here, document what it means or even better calculate at compile-time.